models.py

```diff
@@ -307,9 +307,12 @@ def get_task_state(self):
 
     def get_default_impact_title(self):
         layer_name = '%s on %s' % (
```

Comment:
Title needs to be translated as well if possible. So, use format string with context on it, such as:
```suggestion
        layer_name = '{hazard} on {exposure}'.format(
            hazard=self.hazard_layer.title,
            exposure=self.exposure_layer.title)
```